Modules/Devices/test/Devices.Domain.Tests/Identities/StartDeletionProcessBySupportTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Backbone.BuildingBlocks.Domain;
+using Backbone.DevelopmentKit.Identity.ValueObjects;
+using Backbone.Modules.Devices.Domain.Aggregates.Tier;
+using Backbone.Modules.Devices.Domain.Entities.Identities;
+using Backbone.Modules.Devices.Domain.Tests.Identities.TestDoubles;
+using Backbone.Tooling;
+using FluentAssertions;
+using Xunit;
+
+namespace Backbone.Modules.Devices.Domain.Tests.Identities;
+
+public class StartDeletionProcessBySupportTests
+{
+    [Fact]
+    public void Start_deletion_process_as_support()
```

Comment:
Since all the tests in this file are for "as support", you can remove that part from the test names